### PR TITLE
feat(compass-schema,compass-schema-validation): more darkmode and maps fixes COMPASS-5686, COMPASS-6553, COMPASS-6569

### PR DESCRIPTION
**Pull request by mcasimir**
- Fix spacings with nested docs
- darkMode map background
- remove width calculation for the map and uses a more appropriate height that doesn't cause the background to show up
- setup a listener on `will-navigate` to properly open all of the links outside of electron
- replace `palette.white` with `gray.light1`, `gray.light2` where is too bright
- make the copyright dynamic

### CODE REVIEW

File: packages/compass-schema/src/components/coordinates-minichart/utils.js [hunk 1]
```diff
@@ -46,7 +48,7 @@ const getHereAttributionMessage = async function (bounds, level) {
   });
 
   const copyrightString = copyrights.join(', ');
-  return ` &copy; 1987-2019 HERE${
+  return ` &copy; 1987-${thisYear} HERE${
     copyrightString.length > 0 ? `, ${copyrightString}` : ''
   } | <a href="https://legal.here.com/en/terms/serviceterms/us">Terms of Use</a>`;
```
Comment (gribnoysup) [inline packages/compass-schema/src/components/coordinates-minichart/utils.js:53]: We probably want this too

```suggestion
  } | <a target="_blank" href="https://legal.here.com/en/terms/serviceterms/us">Terms of Use</a>`;
```

File: packages/compass/src/main/window-manager.ts [hunk 2]
```diff
@@ -170,6 +170,11 @@ function showConnectWindow(
     return { action: 'deny' };
   });
 
+  window.webContents.on('will-navigate', function (e, url) {
+    e.preventDefault();
+    void shell.openExternal(url);
+  });
```
Comment (gribnoysup) [inline packages/compass/src/main/window-manager.ts:176]: The less we have of this kind of stuff, the better, we already handle opening windows with `openExternal` in `setWindowOpenHandler`, seems like we are in control of the link there, so might be better to just add a blank target there

Comment (gribnoysup) [reply packages/compass/src/main/window-manager.ts:176]: I left a comment in COMPASS-5686, adding this handler just hides the issue that will also affect DE and I don't think hiding it (either by auto opening in external window or by logging) is a way to go here. If we want to add a listener, then it should be done in a way that will make sure we notice that we added a code that creates an issue

Comment (mcasimir) [reply packages/compass/src/main/window-manager.ts:176]: I mean I understand the idea and I agree, but we still should have a different default. In my opinion, leaving this to "we should remember to add _target='blank'" is not enough, we need a bit more of safety net.

Comment (gribnoysup) [reply packages/compass/src/main/window-manager.ts:176]: We discussed f2f and decided to not automatically open external here and just prevent link from opening completely and debug log a warning that links should have a target blank to be able to open them through compass

**Review by gribnoysup — APPROVED** [review]

**Pull request merged**